travis/samples/ms-ngx-gateway-consul-clustered/.yo-rc.json

```diff
@@ -7,7 +7,7 @@
     "authenticationType": "jwt",
     "cacheProvider": "hazelcast",
     "enableHibernateCache": true,
-    "clusteredHttpSession": true,
+    "clusteredHttpSession": "hazelcast",
```

Comment:
I introduced a wrong config here, that means the `clusteredHttpSession` wasn't test since few days and it seems currently broken
See the Travis build: https://travis-ci.org/hipster-labs/jhipster-travis-build/jobs/319551371
And the log:
```
Tests in error: 
  WebConfigurerTest.testStartUpDevServletContext:91 » NullPointer
  WebConfigurerTest.testStartUpProdServletContext:77 » NullPointer
```

Maybe I should open a new issue

Reply:
Fixed by https://github.com/jhipster/generator-jhipster/pull/6903